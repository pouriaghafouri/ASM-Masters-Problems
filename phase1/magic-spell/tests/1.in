2 nyeh_HEH0heh
