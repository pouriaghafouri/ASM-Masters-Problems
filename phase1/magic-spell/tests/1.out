heh0HEH_heyn
